server/server.js

```diff
@@ -75,11 +75,11 @@ app.use(corsMiddleware);
 app.options('*', corsMiddleware);
 
 // Body parser, cookie parser, sessions, serve public assets
-
+app.use('/translations', Express.static('translations/locales/'));
```

Comment:
The path needs to point to the location of the *copied* translations...

`app.use('/translations', Express.static(path.resolve(__dirname, '../dist/static/translations'));`

Reply:
@andrewn given your suggested fix here, I think this line may be unnecessary, given [L79](https://github.com/processing/p5.js-web-editor/pull/1491/files/78f87b6ec17c7546ecb841c870bccab59c70ca28#diff-ea75b530d74f94670cea053feb8b961bR79), which serves everything in `/dist/static` to `/`, which I think would include subfolders. Definitely double check and test this though!

Reply:
@catarak, you're right that L79 will serve `/dist/static/translations` but it will sent with a max-age of 1 day. Everything bundled by WebPack uses the file's content hash in the filename (e.g. `app.c41b13c86f9f5577f9cc.js`). The browser can cache this for a long time (in principle, forever) since it'll never change. If the bundle's updated, it has a new filename and the browser fetches the file at the new URL.

For now, we're not versioning the translations assets using a content hash in the filename so we should serve them from `/dist/static/translations` **but** with a different cache strategy. Using an E-Tag with no `max-age` means that the browser can make a quick conditional GET request to check if the file's been updated. If it hasn't been updated, no data is received but if it has, it will be sent the latest file.

In future we should probably use the same content-hash-in-filename strategy for these files too, but it's a bit more complex so this is a fix in the short-term that gets staging working again.

@oruburos this line (L78) needs to be:

```js
app.use(
  '/locales',
  Express.static(
    path.resolve(__dirname, '../dist/static/locales'),
    {
      // Browsers must revalidate for changes to the locale files
      // It doesn't actually mean "don't cache this file"
      // See: https://jakearchibald.com/2016/caching-best-practices/
      setHeaders: res => res.setHeader('Cache-Control', 'no-cache')
    }
  )
);
```